spyder/app/mainwindow.py

```diff
@@ -1049,11 +1049,12 @@ def create_edit_action(text, tr_text, icon):
         self.register_shortcut(doc_action, "_",
                                "spyder documentation")
 
-        if self.help is not None:
-            tut_action = create_action(self, _("Spyder tutorial"),
-                                       triggered=self.help.show_tutorial)
-        else:
-            tut_action = None
+        spyder_vid = ("https://www.youtube.com/playlist"
+                      "?list=PLPonohdiDqg9epClEcXoAPUiK0pN5eRoc")
+        vid_action = create_action(self, _("Tutorial Videos"),

```

Comment:
```suggestion
        vid_action = create_action(self, _("Tutorial videos"),
```

For consistency, this should use sentence case, as does every other action in this menu (besides, inexplicably, `Shortcut Summary`) and (most...) other places in Spyder.